application/controllers/collection_instrument.py

```diff
@@ -500,21 +488,15 @@ def get_instrument_data(instrument_id, session):
         file_name = None
 
         if instrument:
-            if instrument.seft_file.gcs:
-                try:
-                    survey_ref = get_survey_ref(instrument.survey.survey_id)
-                    exercise_id = str(instrument.exids[0])
-                    file_path = survey_ref + "/" + exercise_id + "/" + instrument.seft_file.file_name
-                    seft_ci_bucket = GoogleCloudSEFTCIBucket(current_app.config)
-                    file = seft_ci_bucket.download_file_from_bucket(file_path)
-                    return file, instrument.seft_file.file_name
-                except Exception:
-                    log.exception("Couldn't find SEFT CI in GCP bucket")
-            else:
-                log.info("Decrypting collection instrument data", instrument_id=instrument_id)
-                cryptographer = Cryptographer()
```

Comment:
Just looking at the repo after this change, the cryptographer isn't used anymore.  Can you remove it and the stuff that goes with it as well?

Reply:
You can remove the ons_cryptokey value too from the config and helm chart as well.  I think the secret might have to stay where it is as frontstage is using it I think, but I could be wrong about that.